index.html

```diff
@@ -178,21 +178,39 @@ <h2>About Me</h2>
               I'm always interested in hearing about new projects, so if you'd
               like to chat please get in touch.
             </p>
-            <form action="" id="form-contact">
-              <input class="form-text" type="text" placeholder="Full name" />
+            <form
+              id="form-contact"
+              method="POST"
+              action="https://formspree.io/f/xlekdbpd"
+            >
+              <input
+                class="form-text"
+                type="text"
+                name="name"
+                placeholder="Full name"
+                maxlength="50"
```

Comment:
- [ ] good job implemting the contact form :clap:, however please notice that as mentioned in the project requirements we have to make sure Name field accepts up to 30 chars, I would suggest to update the maxlength attribute to 30.